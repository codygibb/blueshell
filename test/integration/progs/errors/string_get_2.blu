s := "hello"
print s[-1]
