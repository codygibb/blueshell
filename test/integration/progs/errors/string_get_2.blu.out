error: Index_out_of_bounds -1
